static/src/js/components/SecondaryToolbar/SecondaryToolbar.js

```diff
@@ -106,16 +107,18 @@ class SecondaryToolbar extends Component {
     let { p = '' } = params
     p = p.replace(/^[^!]*/, '')
 
+    const newSearch = stringify({
+      ...params,
+      p
+    })
     const backLink = (
       <PortalLinkContainer
         className="collection-results__item-title-link"
         to={{
           pathname: '/search',
-          search: stringify({
-            ...params,
-            p
-          })
+          search: newSearch
         }}
+        onClick={() => { onChangePath(`/search/${newSearch}`) }}
```

Comment:
Minor, but we cleaned up the trailing slash everywhere else, should do the same here.